apis/rds/v1alpha1/custom.go

```diff
@@ -0,0 +1,26 @@
+package v1alpha1
+
+import "github.com/crossplane-contrib/terrajet/pkg/terraform/resource"
+
+// DBClusterExternalNamer is config for external name mechanism of DBCluster.
+var (
+	DBClusterExternalNamer = resource.ExternalNameConfiguration{
+		SelfVarPath: "github.com/crossplane-contrib/provider-tf-aws/apis/rds/v1alpha1.DBClusterExternalNamer",
+		Configure: func(base map[string]interface{}, name string) {
+			base["cluster_identifier"] = name
```

Comment:
nit: 
`name` parameter here is the external name of the resource right? Would be more obvious if the parameter was named as `externalName` or maybe `id`.